lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedTaskEventHandler.cs

```diff
@@ -91,6 +92,11 @@ public void OnNext(IAllocatedEvaluator value)
             public void OnNext(IFailedTask value)
             {
                 Logger.Log(Level.Error, FailedTaskMessage);
+                if (value.Message == null || value.Message != ExpectedExceptionMessage)
```

Comment:
`==` is the same as `.Equals` for `string` comparison. It performs an actual lexical comparison rather than a reference comparison.